### ISSUE TITLE
Agent of the Imperium: QA missing errors

### ISSUE THREAD

**Issue by KevinFournier** [RosterizerTestData/Warhammer40k10e#128]
## Exaction Squad:

- The limit of 2 weapon replacement per 5 models in squad does not works. (see bullet point image below). 

![image](https://github.com/RosterizerTestData/Warhammer40k10e/assets/11573588/4eb0fe17-317b-4021-bd98-6a4c8ff317cd)

## Vigilant Squad

- You can take more than 2 special weapon as replacement of Arbites Combat Shotgun.
- You can take more than once the same special weapon as replacement of Arbites Combat Shotgun.

## Inquisitorial Henchmen

- In the unit selection, this unit show a cost of 0 pt instead of 40 (see below)
![image](https://github.com/RosterizerTestData/Warhammer40k10e/assets/11573588/8b938764-ee51-4f86-b3e7-3743ef21c5ab)

- Adding a Gun Servitor or a Mystics add 10 to the unit price. Making the price 50 by adding only one of them. Adding only one of them should cost 20 flat and the other should then not augment the price. The line in the munitorium is `4 Inquisitorial Acolytes, 1 Gun Servitor and 1 Mystic: 60pt`, while `4 Inquisitorial Acolytes: 40pt`.

Basically, in rosterizer, whatever model is added to the unit, it add 10 to the unit price.

To be honest, this one is weird because the prices related to the composition does not cover all composition possible as describe in the datasheet (the 10pt per model seems like this is what it should be 🤷). See below:

Munitorium:
![image](https://github.com/RosterizerTestData/Warhammer40k10e/assets/11573588/08fe2829-70d5-472b-a41b-198edb152f0a)

Datasheet:
![image](https://github.com/RosterizerTestData/Warhammer40k10e/assets/11573588/fe6b11b5-e17c-4569-9171-0c1ce135751a)

**Comment by Nemo3823** [comment]
Fixed in 10.3.0, I've made henchman display as 40, but as the person who did the points rule said, it is just a little bit too annoying to make this work out correctly.
